### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by 94noni**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | maybe |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

The readme is more explicit  like that IMO.
It is strange to talk about downloading Symfony Standard Edition and on the link speaking about the installer

Moreover, where we can update this doc part ?
capture d ecran 2015-04-14 a 10 49 20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,8 +33,7 @@ work for you:
 Installation
 ------------
 
-The best way to install Symfony is to download the Symfony Standard Edition
-available at [http://symfony.com/download][1].
```
Comment (94noni) [inline README.md]: don't know if I should remove the [1] url and re init links incrementation

**Comment by javiereguiluz** [comment]
@94noni I like your proposal to update the README file and recommend the installer. I think it's the last place where we don't recommend it yet, so thanks for spotting it.

However, I don't think it's a good idea to change the `2.3` version number for `lts`. If you are a Symfony newcomer, seeing `lts` may be very confusing: a lot of people don't know what `lts` stands for. And even worse, the lowercase L letter can be confussed with a `1` number and a `i` lowercase letter. That's why I think the current messages are more clear to newcomers (which are the only ones who use this page).

**Comment by 94noni** [comment]
@javiereguiluz ok for your second note, but does this page info (the 2.3 text) changes automatically ? (I mean when 2.7 will be out) because if not, this is confusing :s

**Comment by javiereguiluz** [comment]
@94noni it's not 100% automatic, but we'll change it timely when 2.7 (and 3.0) is released.

**Comment by 94noni** [comment]
Ok, great

File: README.md [hunk 2]
```diff
@@ -33,8 +33,7 @@ work for you:
...
+The best way to install Symfony is to use the [official Symfony Installer][8]. It allows you to start a new project based on the version you want.
```
Comment (fabpot) [inline README.md]: Can you wrap at 80 characters?

Comment (94noni) [inline README.md]: sure let me do that

**Comment by xabbuh** [comment]
:+1:

**Comment by dunglas** [comment]
I prefer the http://symfony.com/download as the first link instead of the book entry.
It more attractive (JS, access to standalone components on the same page...).

**Comment by nicolas-grekas** [comment]
@94noni can you please rebase (we changed all URLs to httpS) and squash also?

**Comment by 94noni** [comment]
I'm gonna close and reopen a new one (I used the github editor for this mini PR)

**Comment by 94noni** [comment]
Closed, see https://github.com/symfony/symfony/pull/14646

**Pull request closed without merging**